### PR TITLE
Two progress bars

### PR DESCRIPTION
**Pull request by Jaqueeee**
Separate progress bars for daemon and wallet

Also removed the old "sync local daemon while using a remote node" logic which has become obsolete after #1091

### CODE REVIEW

**Comment by Jaqueeee** [comment]
skarmavbild 2018-03-15 kl 16 59 47
skarmavbild 2018-03-15 kl 17 01 18

File: components/ProgressBar.qml [hunk 1]
```diff
@@ -44,24 +45,19 @@ Rectangle {
...
         if(targetBlock > 0) {
-            var remaining = targetBlock - currentBlock
-            // wallet sync
-            if(blocksToSync > 0)
-                var progressLevel = (100*(blocksToSync - remaining)/blocksToSync).toFixed(0);
-            // Daemon sync
-            else
-                var progressLevel = (100*(currentBlock/targetBlock)).toFixed(0);
+            var remaining = (currentBlock < targetBlock) ? targetBlock - currentBlock : 0
+            var progressLevel = (blocksToSync > 0) ? (100*(blocksToSync - remaining)/blocksToSync).toFixed(0) : 100
```
Comment (moneromooo-monero) [inline components/ProgressBar.qml]: In these two lines, the difference between targetBlock and blocksToSync isnt't clear. For the progress level to match remaining, they shoiuld be the same, no (plus or minus one) ?

Comment (Jaqueeee) [reply components/ProgressBar.qml]: It's related to #387. If the difference between currentBlock and targetBlock is small, the progress bar would be close to 100% filled during the whole sync progress. 

blocksToSync is set at wallet startup and the progress bar indicated the sync progress from 0% to 100% in the current session. If you restart the wallet, the bar restarts from 0%.

Comment (moneromooo-monero) [reply components/ProgressBar.qml]: OK, that makes sense.

Comment (skftn) [reply components/ProgressBar.qml]: I have a bug where the daemon progress is always 100%. Possibly happens when `blocksToSync` is undefined, see how its used in `main.qml`.

**Pull request merged**